### PR TITLE
add new keymap (xd75 atomic style japanese layout)

### PR DESCRIPTION
**Pull request by takashiski**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xd75/keymaps/atomic_style_jp/keymap.c [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            #ifdef BACKLIGHT_ENABLE
+              backlight_step();
+            #endif
+          } else {
+            unregister_code(KC_RSFT);
+          }
+        break;
+      }
+    return MACRO_NONE;
+};
```
Comment (ridingqwerty) [inline keyboards/xd75/keymaps/atomic_style_jp/keymap.c]: This whole section appears to be unused (due to absence of `M(0)` in keymap), and in any case this function is deprecated.  This is the TMK-style macro functionality; QMK has moved to using `process_record_user`.  Please see [this](https://github.com/qmk/qmk_firmware/blob/master/docs/feature_macros.md#the-new-way-send_string--process_record_user).

I would remove this whole section.

Comment (takashiski) [reply keyboards/xd75/keymaps/atomic_style_jp/keymap.c]: I removed the section

File: keyboards/xd75/keymaps/atomic_style_jp/keymap.c [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+ // 2019/03/04 takashiski
+ // NORTICE
+ // This layout is for Japanese Keyboard User.
+ // So there are logicalparing symbols and special keycode like a KC_RO, KC_HENK, KC_MHEN, KC_JYEN.
+ // 日本語キーボードとして認識させて使うことが前提なので、記号位置が英語キーボードとは異なります。
+ // また、日本語キーボードでないと認識されない特殊なキーコードが含まれています。
+
+#include "xd75.h"
```
Comment (drashna) [inline keyboards/xd75/keymaps/atomic_style_jp/keymap.c]: This should be replaced with: 
```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/xd75/keymaps/atomic_style_jp/keymap.c [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define ___T___ KC_TRNS
+#define XXXXXXX KC_NO
```
Comment (drashna) [inline keyboards/xd75/keymaps/atomic_style_jp/keymap.c]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31

The middle one, though, isn't present, but shouldn't really be used, either.

File: keyboards/xd75/keymaps/atomic_style_jp/config.h [hunk 4]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2017 Benjamin Kesselring
```
Comment (fauxpark) [inline keyboards/xd75/keymaps/atomic_style_jp/config.h]: This file can be deleted.

File: keyboards/xd75/keymaps/atomic_style_jp/keymap.c (hunk 3 above)
Comment (takashiski) [reply keyboards/xd75/keymaps/atomic_style_jp/keymap.c]: I removed their!

**Comment by takashiski** [comment]
I fixed.

**Pull request merged**